.github/workflows/ci-build.yaml

```diff
@@ -407,6 +407,7 @@ jobs:
           # port 8080 which is not visible in netstat -tulpen, but still there
           # with a HTTP listener. We have API server listening on port 8088
           # instead.
+          make controller repo-server server applicationset-controller
```

Comment:
Is this necessary? I think the next line starts everything from source code. I think this would only make sense if BIN_MODE were `true`.

Reply:
This compiles and caches the Go code sequentially before starting the e2e procedure. 

Goreman does this all in parallel, saturating the GitHub runner to a point of OOM/crash.

Reply:
as jann mentioned ci checks weren't running and required this changes.